velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include <algorithm>
```

Comment:
Why is this needed ?

Reply:
The `min()` function used below comes from the standard algorithms library.